packages/shared_preferences/shared_preferences_macos/LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
Shouldn't this be the Chromium license?

See https://github.com/flutter/plugins/blob/master/packages/shared_preferences/shared_preferences/LICENSE

Reply:
/cc @stuartmorgan

Reply:
All licensing should match this repo. FDE used a different license in general, which is why the plugin did.

As the author, I confirm that this can be relicensed to match this repository.

Reply:
Great, thanks!